### ISSUE TITLE
How to convert indexed page numbers to links back to the page

### ISSUE THREAD

**Issue by Webcompose** [asciidoctor/asciidoctor#3711]
If this is already possible add that to the manual for asciidoctor

**Comment by Mogztter** [comment]
Could you please provide an example with a snippet of text to demonstrate what you want to do?

**Comment by Webcompose** [comment]
Say for example I place double brackets around a word on page 31:
The double brackets of a ((candidate)) for the index produces the following:

and if you have in the backmatter 
[index]
== Index

The index will give

**Index**
**C**
Canada, 22
candidate, 31

What I want it to is add a hyperlink to the 31 so that it brings me to page 31 to see how candidate is used
and to do the same for all indexed items

**Comment by Mogztter** [comment]
Thanks, it makes sense.
Which converter are you using? How do you produce the output?

**Comment by Webcompose** [comment]
Thanks for the help. Your suggestion was correct and I see it was covered but never added to the manual.
I think the issue should be closed after adding the instructions to the manual.:
I changed 
:media:prepress
     to
:media: screen in the index.asc as you suggested
and that solved it

**Comment by mojavelinux** [comment]
It seems like you are referring to PDF output. Can you confirm that's true?

**Comment by Webcompose** [comment]
Yes. the whole operation was in my Gitlab project that took advantage of the CI/CD devops feature that will output several formats including HTML, mobi, ePub and PDF
But it was the PDF conversion that needed the screen feature turned on in the index.asc

**Comment by mojavelinux** [comment]
> But it was the PDF conversion that needed the screen feature turned on in the index.asc

That's what I needed to know.

I'm going to move this issue to asciidoctor-pdf as it pertains to that converter.
